src/main/java/com/microsoft/graph/requests/extensions/WorkbookChartSeriesCollectionRequestBuilder.java

```diff
@@ -21,7 +21,7 @@
 import com.microsoft.graph.requests.extensions.WorkbookChartSeriesItemAtRequestBuilder;
 import com.microsoft.graph.http.BaseCollectionRequestBuilder;
 import com.microsoft.graph.core.IBaseClient;
-
+import com.microsoft.graph.http.PrimitiveRequestBuilder;
```

Comment:
This probably shouldn't have this import as it isn't used and I don't think it makes sense here.

Reply:
I don't know why it is here, although there is a count method composed

Reply:
fixed. I wanted to keep the template simpler but inserting references when not needed can be confusing. Plus it has a (limited) impact on build performance for us